src/libp2p_ipc/libp2p_ipc.capnp

```diff
@@ -332,7 +332,7 @@ struct Libp2pHelperInterface {
     result @1 :ValidationResult;
   }
 
-  struct DeleteResource {
+  struct RemoveResource {
```

Comment:
A bit odd that we use tag below but not here, do you have an answer why?

Reply:
tag is a bit opaque I agree, it is mainly used in the mina net 2 library. The purpose seems to be to detect the type of body to sync. For example you may have a different action enabled for downloading epoch ledger vs block.

Reply:
_UPD_ tag is part of encoded data, and root block id is hash of something derived from that data.

I.e. tag is sometimes provided to API for purposes of differentiating which function to use for the data. But for removal it isn't needed, data is perfectly referenced by the root block id alone.